source/install/deprecated-features.rst

```diff
@@ -6,6 +6,13 @@ This page describes features that are removed from support for Mattermost, or wi
 Removed features in upcoming versions
 -------------------------------------
 
+Mattermost Server v9.0.0
+~~~~~~~~~~~~~~~~~~~~~~~~
+
+- Mattermost Boards will transition to being fully community supported, with various other plugins following suit. See this `forum post <https://forum.mattermost.com/t/upcoming-product-changes-to-boards-and-various-plugins/16669>`_ for more details.
+- Removing deprecated Insights feature.
+- Gfycat as a GIF service is being `discontinued <https://techcrunch.com/2023/07/05/gfycat-shuts-down-on-september-1/>`_ on September 1, 2023. The in-product GIF picker currently powered by Gfycat will be replaced with GIPHY in the 9.0 release and in our `supported versions <https://docs.mattermost.com/upgrade/release-lifecycle.html>`_.
```

Comment:
@johndavidlugtu Should we add a link to our blog post on this for more context on what they can expect?

Reply:
Once we have the post live, I think that's a good idea. We just need to confirm which release versions we'll be shipping the fix and timeline first